functional-samples/sample.tabcapture-recorder/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Should we add 116 as the  "minimum_chrome_version"?

Reply:
Done! Great suggestion, thanks.